exp/services/recoverysigner/cmd/keyset.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"go/types"
+	"strings"
+
+	"github.com/google/tink/go/hybrid"
+	"github.com/google/tink/go/insecurecleartextkeyset"
+	"github.com/google/tink/go/integration/awskms"
+	"github.com/google/tink/go/keyset"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/support/config"
+	supportlog "github.com/stellar/go/support/log"
+)
+
+type KeysetCommand struct {
+	Logger         *supportlog.Entry
+	RemoteKEKURI   string
+	OutputFilePath string
+}
+
+func (c *KeysetCommand) Command() *cobra.Command {
+	configOpts := config.ConfigOptions{
+		{
+			Name:        "remote-kek-uri",
+			Usage:       "URI for a remote key-encryption-key (KEK) used to encrypt Tink keyset",
+			OptType:     types.String,
+			ConfigKey:   &c.RemoteKEKURI,
+			FlagDefault: "",
+			Required:    false,
+		},
+	}
+	cmd := &cobra.Command{
+		Use:   "keyset",
```

Comment:
It's a little wordy but if we name this `encryption-tink-keyset` it will be hyper ultra clear that the subcommands under this command relate to the keyset configured with `--encryption-tink-keyset`.